.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,6 +37,8 @@ In order to streamline the review of the contribution we ask you follow these gu
   mvn site
   ```
 
+- [ ] Have you ensured that any documentation diagrams have been updated, along with their source files, using draw.io? See [Metron Development Guidelines](https://cwiki.apache.org/confluence/display/METRON/Development+Guidelines) for instructions.
```

Comment:
Can you make draw.io a link `[draw.io](https://www.draw.io/)`, just to make it super easy for anyone who's never heard of / used it before?

Reply:
@justinleet, yeah that makes sense. Done.